### ISSUE TITLE
There is no default targets@tauri

### ISSUE THREAD

**Issue by Andrysky** [gitbutlerapp/gitbutler#5514]
### Version

0.13.13

### Operating System

macOS

### Distribution Method

dmg (Apple Silicon)

### Describe the issue

I downloaded gitbutler, opened it, selected the folder where the existing git project is located - I get:

![image](https://github.com/user-attachments/assets/b372d592-3fb6-46a7-b8a1-3d6ad85c8f81)

### How to reproduce

select folder with git project

### Expected behavior

error not show

### Relevant log output

```shell
5967ab50b93645e2983b08b0293b4a6b: There is no default target s@tauri://localhost/_app/immutable/chunks/ipc.G7YYbRw6.js:1:734 fromError@tauri://localhost/_app/immutable/chunks/ipc.G7YYbRw6.js:1:903 @tauri://localhost/_app/immutable/chunks/ipc.G7YYbRw6.js:1:1111

```
```

**Comment by estib-vega** [comment]
Taking a look

**Comment by estib-vega** [comment]
@Andrysky Thanks for reporting!
This is highly appreciated :)

**Comment by krlvi** [comment]
This should now be fixed with release 0.13.14 https://github.com/gitbutlerapp/gitbutler/releases/tag/release%2F0.13.14
Let me know if it is resolved for you @Andrysky

**Comment by Andrysky** [comment]
updated - I don't see the problem)
